Formula/linux-pam.rb

```diff
@@ -0,0 +1,45 @@
+class LinuxPam < Formula
+  desc "Pluggable Authentication Modules for Linux"
+  homepage "http://www.linux-pam.org"
+  url "https://github.com/linux-pam/linux-pam/releases/download/v1.5.1/Linux-PAM-1.5.1.tar.xz"
+  sha256 "201d40730b1135b1b3cdea09f2c28ac634d73181ccd0172ceddee3649c5792fc"
+  license any_of: ["BSD-3-Clause", "GPL-1.0-only"]
+  head "https://github.com/linux-pam/linux-pam.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "berkeley-db"
+  depends_on "libprelude"
+  depends_on "libtirpc"
+  depends_on :linux
+
+  skip_clean :la, "etc"
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --disable-selinux
+      --prefix=#{prefix}
+      --includedir=#{include}/security
+      --oldincludedir=#{include}
+      --enable-securedir=#{lib}/security
+      --sysconfdir=#{etc}
+      --with-xml-catalog=#{etc}/xml/catalog
+      --with-libprelude-prefix=#{Formula["libprelude"].opt_prefix}
+    ]
+
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+  end
+
+  def post_install
+    chmod "u=rwxs,g=rx,o=rx", "#{sbin}/unix_chkpwd"
+  end
+
+  test do
+    File.exist? "#{sbin}/unix_chkpwd"
```

Comment:
This won't do anything, need to `assert` on it and specify a message.

Reply:
I removed that test as it's useless.

We could write a test that builds against the pam library ... but I think I'll skip that, because writing a pam module looks like ... scary :D